### PR TITLE
use git.key_file for cloning edx-theme-codebase

### PR DESCRIPTION
**Pull request by OmarIthawi**
This an attempt to debug the described in #344. Celery workers no longer fails on the themes step, `edxapp` now fails for undetermined git/ssh issues.

### CODE REVIEW

File: playbooks/roles/edxapp/tasks/customer_theme.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ---
 
 - debug: msg="Checking out site theme {{item}}"
-  with_dict: "{{EDXAPP_SITE_THEMES}}"
+  with_dict:
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/tasks/customer_theme.yml:4]: more debug info

File: playbooks/roles/edxapp/tasks/customer_theme.yml [hunk 2]
```diff
@@ -10,6 +13,7 @@
     repo: "{{ item.value.theme_source_repo }}"
     version: "{{ item.value.theme_version }}"
     accept_hostkey: yes
+    key_file: "{{ edxapp_git_identity }}"
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/tasks/customer_theme.yml:16]: use key_file to hopefully fix git clone permission issue

File: playbooks/roles/edxapp/tasks/customer_theme.yml [hunk 3]
```diff
@@ -25,7 +29,7 @@
     repo: "{{ item.value.customer_theme_source_repo }}"
     version: "{{ item.value.customer_theme_version }}"
     accept_hostkey: yes
-    key_file: "/edx/app/edxapp/.ssh/id_rsa"
+    key_file: "{{ edxapp_git_identity }}"
```
Comment (OmarIthawi) [inline playbooks/roles/edxapp/tasks/customer_theme.yml:32]: this fix is likely to be not needed, but using the variable which is on `/edx/app/edxapp/edxapp-git-identity`.

**Review by thraxil — APPROVED** [review]

**Comment by OmarIthawi** [comment]
:tada: The Juniper draft theme is being deployed to staging! There was an ssh permission issue and now it's fixed :tada:

```
TASK [edxapp : checkout comprehensive themes] **********************************
ok: [104.196.169.111] => (item={'key': 'tahoe-us-juniper-staging.appsembler.com', 'value': {'theme_name': 'edx-theme-codebase', 'theme_source_repo': 'git@github.com:appsembler/edx-theme-codebase.git', 'theme_version': 'matej/juniper', 'customer_theme_source_repo': 'https://github.com/appsembler/edx-theme-customers.git', 'customer_theme_version': 'juniper/tahoe-initial'}})
TASK [edxapp : checkout customer override themes] ******************************
ok: [104.196.169.111] => (item={'key': 'tahoe-us-juniper-staging.appsembler.com', 'value': {'theme_name': 'edx-theme-codebase', 'theme_source_repo': 'git@github.com:appsembler/edx-theme-codebase.git', 'theme_version': 'matej/juniper', 'customer_theme_source_repo': 'https://github.com/appsembler/edx-theme-customers.git', 'customer_theme_version': 'juniper/tahoe-initial'}})
```

**Comment by OmarIthawi** [comment]
:tada: The Juniper draft theme is being deployed to staging! There was an ssh permission issue and now it's fixed :tada:

```
TASK [edxapp : checkout comprehensive themes] **********************************
ok:
TASK [edxapp : checkout customer override themes] ******************************
ok:
```

**Pull request merged**